src/uct/ib/ud/base/ud_iface.c

```diff
@@ -348,6 +348,10 @@ static void uct_ud_iface_timer(int timer_id, int events, void *arg)
 {
     uct_ud_iface_t *iface = arg;
 
+    if (ucs_unlikely(iface->async.disable)) {
```

Comment:
maybe move this check inside uct_ud_iface_async_progress() for both timer and event?

Reply:
done

Reply:
done